airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/VersionCell.tsx

```diff
@@ -106,13 +106,13 @@ const VersionCell: React.FC<IProps> = ({ id, version, onChange, feedback, curren
                 </InputField>
               )}
             </Field>
-            <LoadingButton
+            <Button
```

Comment:
This button maybe needs to be smaller. Also this is what happens when I click on the button, it moves to the next line:
![Screen Shot 2022-09-22 at 15 57 25](https://user-images.githubusercontent.com/168664/191839678-bee0d7b5-b744-4719-8ae5-52f50f371237.png)